FriendsWave/social/models.py

```diff
@@ -1,3 +1,90 @@
+from functools import _Descriptor
 from django.db import models
+from django.db.models.base import Model
+from django.db.models.fields.related import OneToOneField
+
```

Comment:
A light comment(specifying purpose) should be added on each field for each model, and a doc-string(purpose/utility) added to each class of the code